packages/ripple-nuxt-tide/modules/landing-page/lib/content-collection.js

```diff
@@ -992,13 +992,13 @@ module.exports = class ContentCollection {
     let mappedResult = null
     const _source = item._source
     const link = this.getLocalisedLinkFromSource(_source)
-
+    const newsDate = _source.field_news_date
     switch (this.getDisplayResultComponentType()) {
       case 'search-result':
         mappedResult = {
           title: _source.title?.[0],
           link: link ? { linkText: link.domain + link.path, linkUrl: link.path } : null,
-          date: _source.created?.[0],
+          date: newsDate ? _source.field_news_date?.[0] : _source.created?.[0],
```

Comment:
This doesn't read very well as is, instead of a ternary here this could be a [logical OR](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_OR) and use [optional chaining ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) to guard against undefined values  

eg 
```
date: _source.field_news_date?.[0] || _source.created?.[0]
```
This reads as:
if `field_news_date` is defined and the first item in the array is not falsy, use that otherwise use the other value

Even better would be to [destructure](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#binding_and_assignment) `_source` for just the values needed, but thats unrelated to this PR.

Reply:
Thank you @dylankelly. I have updated my pr for your review.